auth/integration_test/src/integration_test.cc

```diff
@@ -671,6 +671,82 @@ TEST_F(FirebaseAuthTest, TestUpdateUserProfile) {
   DeleteUser();
 }
 
+TEST_F(FirebaseAuthTest, TestUpdateUserProfileNull) {
```

Comment:
I think some blank lines might make this test a little more readable...

Reply:
Fair, done